### PR TITLE
Various Bug Fixes for 1.3.0.0

### PR DESCRIPTION
**Pull request by openbots-ff**

### CODE REVIEW

**Review by openbots-ch — DISMISSED** [review]

**Review by openbots-su — APPROVED** [review]

File: OpenBots.Commands.Test/OpenBots.Commands.Engine.Test/app.config [hunk 1]
```diff
@@ -16,12 +16,40 @@
...
-        <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-5.0.0.1" newVersion="5.0.0.1" />
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Threading.Tasks.Extensions" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.2.0.1" newVersion="4.2.0.1" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
```
Comment (openbots-ab) [inline OpenBots.Commands.Test/OpenBots.Commands.Engine.Test/app.config:27]: Please update the version to the latest one.

File: OpenBots.Commands/OpenBots.Commands.Microsoft/app.config~merged [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
```
Comment (openbots-ab) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/app.config~merged:7]: Please update the version to the latest one as it's updated in the packages.config file.

File: OpenBots.Studio/Nuget/AppDomainSetupManager.cs [hunk 3]
```diff
@@ -40,6 +40,7 @@ public static ContainerBuilder LoadBuilder(List<string> assemblyPaths)
 
             ContainerBuilder builder = new ContainerBuilder();
 
+            var assemblies2 = AppDomain.CurrentDomain.GetAssemblies();
```
Comment (openbots-ab) [inline OpenBots.Studio/Nuget/AppDomainSetupManager.cs:43]: Can you please remove this line as it is not being used?

Comment (openbots-ff) [reply OpenBots.Studio/Nuget/AppDomainSetupManager.cs:43]: done

**Review by openbots-su — APPROVED** [review]

**Review by openbots-ab — APPROVED** [review]

**Pull request merged**